api/src/services/products/products.js

```diff
@@ -6,18 +6,20 @@ export const products = async ({ type = 'one_time' }) => {
     expand: ['data.product'],
   })
 
-  const products = prices.data.map((price) => {
-    const { product } = price
+  const products = prices.data
+    .filter(({ product }) => product.active)
+    .map((price) => {
+      const { product } = price
```

Comment:
@paulasjes-stripe Is there a way to get only prices with active products with the `stripe.prices.list`? 

Right now I get a complete list of prices (filtered by type) from Stripe and then filter that list for prices with active products

Reply:
You can only filter on `active` for the resource in question itself. Instead I'd reverse this and first retrieve a list of active products, then for each retrieve a list of active prices

Reply:
I'll try that. Thanks!